metecho/api/models.py

```diff
@@ -489,6 +566,37 @@ def get_collaborator(self, gh_uid: str) -> Optional[Dict[str, object]]:
             return None
 
 
+class ProjectDependency(HashIdMixin, TimestampsMixin):
+    name = StringField()
+    url = models.URLField()
+    recommended = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = _("project dependency")
+        verbose_name_plural = _("project dependencies")
+        ordering = ("name",)
```

Comment:
Added explicit ordering to avoid potential bugs:

```
metecho/api/tests/views.py::TestProjectDependencyViewset::test_list
  /usr/local/lib/python3.10/site-packages/rest_framework/pagination.py:200: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'metecho.api.models.ProjectDependency'> QuerySet.
```

Reply:
Makes sense -- we don't have a front-end mockup for paginating the list of dependencies, but we can discuss that.